src/app/environments/create-form/fieldsConfiguration.json

```diff
@@ -327,6 +327,23 @@
     "value": "",
     "dependencies": ["stack"],
     "text": true,
```

Comment:
`"text": true` means that this field will be displayed as input text field. This line should be removed to have multiple checkboxes for it.